lib/apartment/railtie.rb

```diff
@@ -30,6 +30,14 @@ class Railtie < Rails::Railtie
       Apartment::Tenant.reload!
     end
 
+    config.after_initialize do |app|
+      # NOTE: Load the custom log subscriber if enabled
+      if Apartment.active_record_log
+        ActiveSupport::Notifications.unsubscribe 'sql.active_record'
+        Apartment::LogSubscriber.attach_to :active_record
+      end
+    end
```

Comment:
This is good! It will reduce possible breakage surface for users without the logging.